sdk/src/client/secret/types.rs

```diff
@@ -32,6 +31,17 @@ pub struct StrongholdDto {
     /// The path for the Stronghold file
     pub snapshot_path: String,
 }
+
+/// A password wrapper that takes care of zeroing the memory when being dropped.
+#[derive(Clone, Eq, PartialEq, Serialize, Deserialize, Zeroize, ZeroizeOnDrop, derive_more::From)]
+pub struct Password(String);
```

Comment:
I kinda think this should be in `utils`. I understand why `secret` sounds like a compelling place to have such a type but in reality, it has absolutely nothing to do with the content of this module.

Reply:
Mnemonic is there because it contains a `SecretManage` implementation which is the purpose of the `secret` module. Nothing in `secret` requires `Password` to be there so it shouldn't be there.

Reply:
Okay. I agree. So you `utils` it is. Are you okay with a `client/utils/password.rs`?

Reply:
Done